common-app-lib/src/cmr/common_app/api_docs.clj

```diff
@@ -151,23 +151,42 @@
   "Defines the footer of the generated documentation page."
   "</div></body></html>")
 
+(defn renderer
```

Comment:
Have you run `lein with-profile docs generate-docs` in both access-control-app and ingest-app to make sure the documentation is still generated as before without issue?